cypress.config.js

```diff
@@ -0,0 +1,25 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  screenshotsFolder: 'tests/cypress/screenshots',
+  videosFolder: 'tests/cypress/videos',
+  fixturesFolder: 'tests/cypress/fixtures',
```

Comment:
I suspect this file was autogenerated. 
It's missing the entries for `integrationFolder` and `supportFolder`, as in the deleted JSON file.
When I try to run this locally, Cypress automatically creates a top-level `/cypress` folder, with default integration tests.
I quit Cypress, removed this new folder, and added the missing config data to this file, this fixes everything.

Reply:
Thanks for catching this! I did run tests in all environments but haven't looked carefully at the output. False positives are the worst.

Reply:
hmm, have you ran `yarn` command? If you run old cypress on new thing it should not work.
I can see tests passing local (the actually tests)
Logs in bacon also are showing tests: 
<img width="1123" alt="image" src="https://github.com/okta/okta-help/assets/4158731/e21ff54c-b493-4a24-8f20-4ff7f4d65649">

and the same for CircleCI

Reply:
I pulled down the branch, then ran `yarn run cy:open`, as I usually do.

Reply:
In that case, the "old" cypress is not working well with migrated tests. If you run `yarn` to fetch the correct cypress that should fix it.

Reply:
and, the `yarn run cy:run` command just fails, can't find the spec files